### ISSUE TITLE
Unable to make requests

### ISSUE THREAD

**Issue by whtlnv** [DKunin/optimizilla-cli#4]
The module was installed globally as instructed, but no matter what image I pass to it the response is always:

```
Error: Promise rejected with value: { body: '<!DOCTYPE html>\n<html>\n  <head><title>Server error</title></head>\n  <style>\n      body { background-color: #caecf6 }\n      #raptor {\n        background: url(/mojo/failraptor.png);\n        height: 488px;\n        left: 50%;\n        margin-left: -371px;\n        margin-top: -244px;\n        position:absolute;\n        top: 50%;\n        width: 743px;\n      }\n    </style>\n  <body><div id="raptor"></div></body>\n</html>\n<!-- a padding to disable MSIE and Chrome friendly error page -->\n<!-- a padding to disable MSIE and Chrome friendly error page -->\n<!-- a padding to disable MSIE and Chrome friendly error page -->\n<!-- a padding to disable MSIE and Chrome friendly error page -->\n<!-- a padding to disable MSIE and Chrome friendly error page -->\n<!-- a padding to disable MSIE and Chrome friendly error page -->\n',
  uniqPathId: '2d45553abfe29c85',
  randomId: 'a5c0d14995107e3f58c92efba3',
  fileName: 'ad.png' }
    at /usr/local/lib/node_modules/optimizilla-cli/node_modules/loud-rejection/index.js:27:12
    at Array.forEach (native)
    at EventEmitter.<anonymous> (/usr/local/lib/node_modules/optimizilla-cli/node_modules/loud-rejection/index.js:23:24)
    at emitTwo (events.js:125:13)
    at EventEmitter.emit (events.js:213:7)
    at emit (/usr/local/lib/node_modules/optimizilla-cli/node_modules/signal-exit/index.js:77:11)
    at processEmit [as emit] (/usr/local/lib/node_modules/optimizilla-cli/node_modules/signal-exit/index.js:150:5)
```

**Comment by DKunin** [comment]
Looks like some trouble with hashing paths, will look into it, thanks!

**Comment by DKunin** [comment]
@whitelionV Fixed in 99cf3d1c5846e567e85e281839b1586a23a99bf8

**Comment by DKunin** [comment]
Bumped version to 4.0.0, since need for hasha was dropped.
